scss/custom/_modal.scss

```diff
@@ -0,0 +1,73 @@
+//
+// Full Scale Modal
+//
+
+.modal-full {
+  width: 100%;
+  max-width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+
+  .modal-content {
+    // Fixed for older phones
+    display: -webkit-flex; /* stylelint-disable-line */
```

Comment:
The prefixer will actually add all of these when creating the final .css (https://stylelint.io/user-guide/rules/value-no-vendor-prefix/) just make sure to define the standard variation.